<ns0:test-suite xmlns:ns0="urn:model.allure.qatools.yandex.ru" start="1527393225607" stop="1527393225717">
  <name>test_search</name>
  <labels/>
  <test-cases>
    <test-case start="1527393225607" status="passed" stop="1527393225615">
      <name>Test_001.test_hello[1]</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="72578-MainThread"/>
        <label name="host" value="fengyanyandeMacBook-Pro.local"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1527393225623" status="failed" stop="1527393225713">
      <name>Test_001.test_hello[2]</name>
      <failure>
        <message>AssertionError: assert 2 != 2</message>
        <stack-trace>self = &lt;test_search.Test_001 object at 0x109a99780&gt;, a = 2

    @pytest.mark.parametrize('a', [1, 2, 3])
    def test_hello(self, a):
&gt;       assert a != 2
E       assert 2 != 2

scripts/test_search.py:7: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="72578-MainThread"/>
        <label name="host" value="fengyanyandeMacBook-Pro.local"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1527393225715" status="passed" stop="1527393225717">
      <name>Test_001.test_hello[3]</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="72578-MainThread"/>
        <label name="host" value="fengyanyandeMacBook-Pro.local"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
  </test-cases>
</ns0:test-suite>
